recipes/sagelib/meta.yaml

```diff
@@ -0,0 +1,147 @@
+{% set version = "7.5.1" %}
+{% set name = "sagelib" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/sagemath/sage/archive/{{ version }}.tar.gz
+  sha256: 3cd9af3e3bfd7f5cc966b7c36168d04b9a1213aa9a7af3286b5d53d40e7653e3
+  patches:
+    # deps are already available as conda packages
+    - patches/deps.patch
+    # sage patches cython to include sys.path, workaround that
+    - patches/Fix-cython-bug.patch
+    # sage's planarity is 2.x and conda-forge's is 3.x
+    - patches/u1-version-planarity-3.patch
+    # sage doesn't like building in root
+    - patches/root.patch
+    # sage's gsl is linked with openblas, but conda-forge's is not.
+    - patches/gsl.patch
+    # less verbosity so that output < 4MB
+    - patches/less-verbosity.patch
+    # conda's python doesn't have a symlink from lib/python2.7 to lib/python
+    - patches/sage-script.patch
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  skip: true  # [py!=27]
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - toolchain
+    - libtool
+    - automake
+    - pip
+    - setuptools
+    - arb 2.9.*
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.19|0.2.19.*
+    - brial
+    - cliquer
+    - cysignals
+    - cython
+    - ecl
+    - eclib
+    - ecm
+    - givaro
+    - glpk
+    - gsl
+    - iml
+    - jinja2
+    - jupyter_core
+    - lcalc
+    - lrcalc
+    - libflint
+    - libgap
+    - libgd
+    - libpng >=1.6.28,<1.7
+    - linbox
+    - m4ri
+    - m4rie
+    - mpc
+    - mpfi
+    - mpfr 3.*
+    - gmp >=5.0.1,<7
+    - ntl
+    - numpy 1.11.*
+    - pari
+    - pkgconfig
+    - planarity 3.*
+    - ppl
+    - pynac
+    - python
+    - ratpoints
+    - readline 6.2*
+    - rw
+    - singular
+    - six
+    - symmetrica
+    - zn_poly
+  run:
+    - arb 2.9.*
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.19|0.2.19.*
+    - brial
+    - cliquer
+    - cysignals
+    - cython
+    - ecl
+    - eclib
+    - ecm
+    - givaro
+    - glpk
+    - gsl
+    - iml
+    - jinja2
+    - jupyter_core
+    - lcalc
+    - lrcalc
+    - libflint
+    - libgap
+    - libgd
+    - libpng >=1.6.28,<1.7
+    - linbox
+    - m4ri
+    - m4rie
+    - mpc
+    - mpfi
+    - mpfr 3.*
+    - gmp >=5.0.1,<7
+    - ntl
+    - numpy 1.11.*
+    - pari
+    - pkgconfig
+    - planarity 3.*
+    - ppl
+    - pynac
+    - python
+    - ratpoints
+    - readline 6.2*
+    - rw
+    - singular
+    - six
+    - symmetrica
+    - zn_poly
+
+test:
+  import:
+    - sage
```

Comment:
Cannot import sage.all as all the runtime dependencies are not in conda-forge and some runtime dependencies need sagelib to build.